server/src/index.test.ts

```diff
@@ -25,13 +25,12 @@ describe('express server', () => {
     expect(addressInfo.port).toBe(8080)
   })
 
-  it(`should return same index.html from ${env.APP_PUBLIC_URL} as from ${env.APP_BUCKET_URL} (bucket)`, async () => {
+  it(`should return same index.html from proxy ${env.APP_PUBLIC_URL} as from bucket ${env.APP_BUCKET_URL} (minus the chunk hash)`, async () => {
+    const regExp = /\/(\d|main)\.([0-9a-f]{5,40})(\.chunk\.js)/gm
```

Comment:
c'est un test, je ne pense pas qu'on ai besoin d'etre aussi précis sur le nombre de caractère

je propose

```suggestion
    const regExp = /\/(\d|main)\.([0-9a-f]+)(\.chunk\.js)/gm
```

on a pas besoin de capturer l'identifiant du chunk

je propose

```suggestion
    const regExp = /\/(\d|main)\.[0-9a-f]+(\.chunk\.js)/gm
```